client/PhotoModule.jsx

```diff
@@ -52,11 +64,21 @@ class PhotoModule extends React.Component {
   };
 
   fetchPhotosByStyle = async (productId, styleId) => {
-    const response = await fetch(`http://localhost:3000/photos/${productId}/${styleId}`);
-    const data = await response.json();
-    this.setState({
-      photos: data,
-    });
+    try {
+      const response = await fetch(`http://13.56.180.105/photos/${productId}/${styleId}`);
+      if (!response.ok) {
+        throw new Error('photo fetch request failed');
+      }
+      const data = await response.json();
+      this.setState({
+        photos: data,
+      });
+    } catch (e) {
+      console.log(e);
```

Comment:
you might want to switch it to throw (e) for production build rather than console logs

Reply:
I don't want to throw because that would stop the function. But I'll remove the console.log.

Reply:
Cool! Maybe try console.error to differentiate it from like development console logs!